test/test_transforms_v2_refactored.py

```diff
@@ -2571,7 +2582,15 @@ def test_functional_image_correctness(self, kwargs):
     )
     @pytest.mark.parametrize(
         "make_input",
-        [make_image_tensor, make_image_pil, make_image, make_bounding_boxes, make_segmentation_mask, make_video],
+        [
+            make_image_tensor,
+            make_image_pil,
+            make_image,
+            make_bounding_boxes,
+            make_segmentation_mask,
+            make_detection_mask,
```

Comment:
Why test both here? The other tests that were modified seem to only be `test_functional_signature`?